cli/unregister/unregister.go

```diff
@@ -205,24 +206,40 @@ func DeepClean() error {
 		cliutils.RunCmd(nil, "pkill", "-f", "/usr/horizon/bin/anax")
 
 	} else {
-		cliutils.Verbose(msgPrinter.Sprintf("Stopping horizon..."))
-		cliutils.RunCmd(nil, "systemctl", "stop", "horizon.service")
-
-		msgPrinter.Printf("Deleting local horizon DB...")
-		msgPrinter.Println()
-		cliutils.RunCmd(nil, "bash", "-c", "rm -f /var/horizon/*.db")
-		cliutils.RunCmd(nil, "bash", "-c", "rm -Rf /etc/horizon/policy.d/*")
+		if runtime.GOOS == "darwin" {
+			outBytes, errBytes := cliutils.RunCmd(nil, "/bin/sh", "-c", "docker ps | grep horizon1")
```

Comment:
`horizon1` is just one container. There could be multiple containers running at the same time: horizon1, horizon2 etc. Need to figure out which container the hzn command is talking about. This can be determined by "HORIZON_URL" env variable (https://github.com/open-horizon/anax/blob/master/cli/cliutils/cliutils.go#L508)  and the port binding of the container.